core/src/main/java/io/kestra/core/models/tasks/InputFilesInterface.java

```diff
@@ -7,7 +7,12 @@
 
 public interface InputFilesInterface {
     @Schema(
-        title = "The files to create on the local filesystem. It can be a map or a JSON object.",
+        title = "The files to create on the working. It can be a map or a JSON object.",
+        description = """
+            Each file can be defined:
```

Comment:
would be cool to add an example for both cases

Reply:
Examples are on examples and done by the product team later.